app/src/main/java/com/example/githubclone/presentation/main/MainScreen.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.githubclone.presentation.main
+
+import androidx.compose.material.BottomNavigation
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import com.example.githubclone.navigation.bottombarnavigation.BottomNavGraph
+import com.example.githubclone.navigation.bottombarnavigation.BottomScreen
+
+@Composable
+fun MainScreen() {
+    val navController = rememberNavController()
+    Scaffold(bottomBar = {
+        BottomNavBar(navController = navController)
+    }) { paddingValues ->
+        BottomNavGraph(navController = navController, paddingValues = paddingValues)
+    }
+}
+
+@Composable
+fun BottomNavBar(navController: NavHostController) {
+    val navItems = listOf(
+        BottomScreen.Home,
+        BottomScreen.Notifications,
+        BottomScreen.Explore,
+        BottomScreen.Profile
+    )
```

Comment:
Can we `remember` this list? e.g. `val navItems = remember { mutableStateListOf(BottomScreen.Home, ...) }`

Reply:
Yes, that's a good idea